sdk/eventhub/event-hubs/src/util/runtimeInfo.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as os from "os";
```

Comment:
style thing - should these be `import { type, release } from "os"`?

Reply:
Normally I would have, but in this case since I thought the destructured imports were not descriptive enough. I could have aliased them but I didn't see much reason to do that over the above for a node built-in.